server/paymentProviders/paypal/payouts.ts

```diff
@@ -93,7 +94,18 @@ export const checkBatchItemStatus = async (
   switch (item.transaction_status) {
     case 'SUCCESS':
       if (expense.status !== status.PAID) {
-        await createTransactionFromPaidExpense(host, null, expense, null, expense.UserId, 0, 0, 0, item);
+        const fees = {};
+        const fxRate = 1 / (toNumber(item.currency_conversion?.exchange_rate) || 1);
+        if (item.payout_item_fee) {
+          const paymentProcessorFeeInExpenseCurrency = floatAmountToCents(toNumber(item.payout_item_fee.value));
+          fees['paymentProcessorFeeInHostCurrency'] = Math.round(paymentProcessorFeeInExpenseCurrency * fxRate);
+          if (item.payout_item_fee.currency !== expense.currency) {
+            // payout_item_fee is always supposed to be in currency_conversion.to_amount.currency. This is a sanitiy check just in case
```

Comment:
```suggestion
            // payout_item_fee is always supposed to be in currency_conversion.to_amount.currency. This is a sanity check just in case
```